io.nanoboyadvance.NanoBoyAdvance.yml

```diff
@@ -0,0 +1,69 @@
+id: io.nanoboyadvance.NanoBoyAdvance
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: NanoBoyAdvance
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --filesystem=host # for save files
```

Comment:
this is very broad to just "save files"

Reply:
Seems to support filechooser portal, do you still need it?